build/docma-config.json

```
@@ -222,7 +222,7 @@
             "plugins": [
                 "web/client/plugins/index.jsdoc",
                 "web/client/plugins/AddGroup.jsx",
-                "web/client/plugins/Annotations.jsx",
+                "web/client/plugins/Annotations/index.jsx",
```

Wrong file name caused the doc not to be added to the plugins documentation
```suggestion
                "web/client/plugins/Annotations/index.js",
```